lib/flows/login-flow.js

```diff
@@ -49,7 +49,10 @@ export default class LoginFlow {
 		}
 	}
 
-	async login( { jetpackSSO = false, jetpackDIRECT = false, screenshot = false } = {}, eyes ) {
+	async login(
+		{ emailSSO = false, jetpackSSO = false, jetpackDIRECT = false, screenshot = false } = {},
+		eyes
```

Comment:
Maybe you will have some merge conflicts since eyes helper will be removed. I'm not sure is it on master yet, but try to merge master first and see what will happen.